auth/src/api/builder.rs

```diff
@@ -73,6 +74,19 @@ impl ApiBuilder {
             .route("/auth/refresh", post(refresh_token))
             .route("/public-key", get(get_public_key))
             .route("/users/:account_name", get(get_user))
+            .route(
+                "/users/subscription/items",
+                post(
+                    add_subscription_items.layer(JwtAuthenticationLayer::new(move || {
+                        let public_key = public_key.clone();
+                        async move { public_key.clone() }
+                    })),
+                ),
+            )
+            .route(
+                "/users/subscription/:account_name",
```

Comment:
Path param is never used. Can it therefore be simplified?

Reply:
This is a route I forgot to remove, so it can be simplified a lot (deleted :smile:)